numpy/ma/timer_comparison.py

```diff
@@ -94,7 +93,7 @@ def assert_array_compare(self, comparison, x, y, err_msg='', header='',
                 reduced = reduced.tolist()
             if not cond:
                 match = 100-100.0*reduced.count(1)/len(reduced)
-                msg = build_err_msg([x, y],
+                msg = build_err_msg([x, y], 
```

Comment:
```suggestion
                msg = build_err_msg([x, y],
```
Trailing whitespace.